src/cljs/main/broadfcui/main.cljs

```diff
@@ -123,14 +123,8 @@
    {:key :policy
     :render #(react/create-element Policy %)}])
 
-(defn- get-authenticated-nav-bar-items [curator?]
-  (if (or (nil? curator?)
-          (not curator?))
-    #{:workspaces :methods}
-    #{:library :workspaces :methods}))
-
 (defn- top-nav-bar-items [state]
-  (filter (fn [r] (contains? (get-authenticated-nav-bar-items (:curator? @state)) (:key r))) routes))
+  (filter (fn [r] (contains? #{:library :workspaces :methods} (:key r))) routes))
```

Comment:
More compact version of this code:

```clojure
(filter (comp #{:library :workspaces :methods} :key) routes)
```

This also doesn't use `state` any more and doesn't need to be a function.